static/index.html

```diff
@@ -4,7 +4,10 @@
     </head>
     <body>
         Prebid Server is a server-to-server proxy for <a href="http://prebid.org">Prebid.js</a> users.
-        The host is not responsible for the content or advertising delivered through this proxy. For more information, 
-        please contact prebid-server - at - prebid.org.
+
+        <p>
+        	This is a modified version for AYL testing.
+        	hitting broker-preprod.
+        </p>
```

Comment:
Please unstage the modifications  to this file

Reply:
done.